### PR TITLE
Porting CM 2x1 & glass door sprites and del some posters

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

More mapping options! Thanks to this CM PR for the sprites: https://gitlab.com/cmdevs/colonial-warfare/-/merge_requests/2701

As of right now, Minerva, Theseus, and Pillar of Spring have the ported doors to showcase the beauty of these doors.

Open source, baby.

## Why It's Good For The Game

It's for the mappers that want to have more spice into their life, including me. I'm doing this so that RipGrayson can make Big Red have like 10000 variances.

## Changelog
:cl: novaepee, RipGrayson
add: port doors into Minerva, Theseus, and Pillar of Spring
del: some bad posters again
imageadd: more door sprites from CM!
imageadd: change brig door to reflect change, not that anyone cares about brig
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/doors/multi_tile.dm [hunk 1]
```diff
@@ -142,7 +142,7 @@
 
 /obj/machinery/door/airlock/multi_tile/mainship/secdoor
 	name = "\improper Security Airlock"
-	icon = 'icons/obj/doors/mainship/2x1secdoor.dmi'
+	icon = 'icons/obj/doors/mainship/2x1secdoor_glass.dmi'
```
Comment (novaepee) [inline code/game/objects/machinery/doors/multi_tile.dm:145]: In the port, CM's 2x1secdoor_glass.dmi is 95% the same as TGMC's 2x1secdoor.dmi **BUT** CM's 2x1secdoor.dmi is very different. I had to change this and pray no mistakes in porting were made.

Comment (novaepee) [reply code/game/objects/machinery/doors/multi_tile.dm:145]: image

Only these maps are affected.

**Comment by novaepee** [comment]
image

Pictures. forgot to show.

**Review by Kuro020 — APPROVED** [review]

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Where are the typepaths for these

**Comment by novaepee** [comment]
> Where are the typepaths for these

Right now, I'm only porting sprites. If you want me to make doors for these, I'm willing to do so.

File: code/game/objects/machinery/doors/multi_tile.dm [hunk 2]
```diff
@@ -103,6 +106,149 @@
...
 
+/obj/machinery/door/airlock/multi_tile/mainship/generic2
+	name = "\improper Airlock"
+	icon = 'icons/obj/doors/mainship/2x1almayerdoor.dmi'
+	opacity = FALSE
+	glass = TRUE
+
+/obj/machinery/door/airlock/multi_tile/mainship/generic2/glass
+	name = "\improper Glass Airlock"
+	icon = 'icons/obj/doors/mainship/2x1almayerdoor_glass.dmi'
```
Comment (TiviPlus) [inline code/game/objects/machinery/doors/multi_tile.dm:117]: A) dont use numbers in typepaths
B) Why is the base glass

File: code/game/objects/machinery/doors/multi_tile.dm [hunk 3]
```diff
@@ -103,6 +106,149 @@
...
+
+/obj/machinery/door/airlock/multi_tile/mainship/marine/general/smart
+	name = "\improper Smartgunner Preparations"
+	req_access = list(ACCESS_MARINE_SMARTPREP)
+
+/obj/machinery/door/airlock/multi_tile/mainship/marine/general/corps
+	name = "\improper Corpsman Preparations"
+	req_access = list(ACCESS_MARINE_MEDPREP)
+	opacity = FALSE
+	glass = TRUE
```
Comment (TiviPlus) [inline code/game/objects/machinery/doors/multi_tile.dm:144]: this is inherited

File: code/game/objects/machinery/doors/multi_tile.dm [hunk 4]
```diff
@@ -103,6 +106,149 @@
...
+	name = "\improper Squad Preparations"
+	icon = 'icons/obj/doors/mainship/2x1prepdoor.dmi'
+	req_access = list(ACCESS_MARINE_PREP)
+	opacity = FALSE
+	glass = TRUE
+
+/obj/machinery/door/airlock/multi_tile/mainship/marine/requisitions
+	name = "\improper Requisitions Bay"
+	req_one_access = list(ACCESS_MARINE_LOGISTICS, ACCESS_MARINE_CARGO)
+
```
Comment (TiviPlus) [inline code/game/objects/machinery/doors/multi_tile.dm:131]: you dont even set the icon/iconstates here go do it properly im not reviewing this

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by novaepee** [comment]
image
Theseus with new general prep door.

image
Minerva engineering

image
Minerva firing range

image
Pillar of Spring firing range and prep

**Pull request merged**